examples/13_path-alias/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "waku-example",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dev": "waku dev --with-ssr",
+    "build": "waku build",
```

Is it intentional to not have `--with-ssr`?

---

Let's add it and see how it goes.
```suggestion
    "build": "waku build --with-ssr",
```